pkgs/by-name/an/angular-language-server/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  mkYarnPackage,
+  fetchYarnDeps,
+  fetchFromGitHub,
+}:
+
+mkYarnPackage rec {
+  pname = "vscode-ng-language-service";
```

Comment:
The `pname` should probably be the same as the package in the path `pkgs/by-name/../<pname>/package.nix`.

Even if the repository is called `vscode-ng-language-service`, in my opinion `angular-language-server` is a much more representative name.

Reply:
Agreed, that's more akin to what it's registered under on npm: `@angular/language-server`.

Reply:
I was also wondering if it would be better located under `pkgs/development/tools/language-servers`?

Reply:
All new packages that can go in `pkgs/by-name`, must go there. Even if you tried, an automated check would tell you to use `by-name`. Some time in the future all other packages(when applicable) will be moved to `by-name`: https://github.com/NixOS/nixpkgs/pull/211832